components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -215,6 +216,37 @@ func (r *NotebookReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, err
 	}
 
+	// Check if annotations for the Notebook instance imply a requirement for notebook restart.
+	// This annotation is supposed to be added once a ConfigMap used in the notebook is added/updated.
+
+	annotations := instance.GetAnnotations()
+	notebookRestart, ok := annotations[AnnotationNotebookRestart]
+
+	if ok && notebookRestart == "true" {
+
```

Comment:
lets add some info log , so we can debug
```suggestion
log.Info("Annotation restart-pod is set, working on restarting the pod")
```